qa/pull-tester/rpc-tests.py

```diff
@@ -435,27 +438,45 @@ def get_next(self):
             self.num_running += 1
             t = self.test_list.pop(0)
             port_seed = ["--portseed={}".format(len(self.test_list) + self.portseed_offset)]
-            log_stdout = tempfile.SpooledTemporaryFile(max_size=2**16)
-            log_stderr = tempfile.SpooledTemporaryFile(max_size=2**16)
+            log_stdout = tempfile.SpooledTemporaryFile(max_size=2**16, mode="w+")
+            log_stderr = tempfile.SpooledTemporaryFile(max_size=2**16, mode="w+")
             self.jobs.append((t,
                               time.time(),
                               subprocess.Popen((RPC_TESTS_DIR + t).split() + self.flags.split() + port_seed,
                                                universal_newlines=True,
                                                stdout=subprocess.PIPE,
-                                               stderr=subprocess.PIPE)))
+                                               stderr=subprocess.PIPE),
+                              log_stdout, log_stderr))
```

Comment:
Indent

Reply:
Should be fine? `log_stdout`, `log_stderr` are part of the tuple appended to the `jobs` list. They are not part of the `subprocess.Popen()`. Or do you mean something else?